pkg/urbit/noun/events.c

```diff
@@ -71,6 +71,18 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 
+//! North segment image file name.
+static const c3_c nor_nam_c[] = "north.bin";
+
+//! South segment image file name.
+static const c3_c sou_nam_c[] = "south.bin";
+
+//! Control patch file name.
+static const c3_c ctl_nam_c[] = "control.bin";
+
+//! Memory patch file name.
+static const c3_c mem_nam_c[] = "memory.bin";
+
```

Comment:
`_nam` is redundant in each of these, s/b just `nor_c`, `sou_c`, &c.

Reply:
True, but it's nice for global variable names to consist of six characters, which makes them easily identifiable amidst the three-character names that are currently the standard for local variable names. I think the ease of identification is worth the few extra keystrokes.